rpc/src/endpoint/consensus_state.rs

```diff
@@ -384,4 +389,11 @@ mod test {
         };
         assert_eq!(summary.to_string(), "Vote{0:000001E443FD 1262197/00/1(Prevote) 634ADAF1F402 7BB974E1BA40 @ 2019-08-01T11:52:35.513572509Z}");
     }
+
+    #[test]
+    fn deserialization_regressions() {
+        for s in DESERIALIZATION_REGRESSIONS {
+            let _ = VoteSummary::from_str(s).unwrap();
```

Comment:
Can we test here that the parse `VoteSummary` contains the expected data?

Reply:
Sure! :+1: